libjolie/src/jolie/lang/parse/util/VisualizeAST.java

```diff
@@ -0,0 +1,458 @@
+/***************************************************************************
```

Comment:
Where is this class used? Please remove it if unused (we could move it to an external library though maybe... but let's leave it for another patch).